src/render/draw_debug.js

```diff
@@ -121,64 +177,6 @@ function drawTileQueryGeometry(painter, sourceCache, coord: OverscaledTileID) {
     }
 }
 
-function drawDebugTile(painter, sourceCache, coord: OverscaledTileID) {
```

Comment:
No functional change in this file? I can't tell from the way it shows the diff as it was moved upward in this file.

Reply:
No functional change, just moved this function to be closer to the parent function.